AlamofireXMLRPC/XMLRPCNode.swift

```diff
@@ -27,6 +28,7 @@ public struct XMLRPCNode {
             }
         }
         self.xml = xml
+        self.rawXML = xml.xml
```

Comment:
Instead of storing it and make it mutable. It will be better to define it as a computed property
```swift
public var rawXml: String {
     return xml.xml
}

```